### PR TITLE
Add "Getting Started" instructions to the readme

### PR DESCRIPTION
**Pull request by cawallin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,34 @@ Comprehensive documentation can be found [here](http://prestodb.github.io/presto
...
+1. Fork the `presto-admin` repo on GitHub, https://github.com/prestodb/presto-admin.
+2. Clone your fork locally::
+
+    $ git clone git@github.com:your_name_here/presto-admin.git
+
+3. Install your local copy into a virtualenv. Assuming you have virtualenvwrapper installed, this is how you set up your fork for local development::
+
+    $ mkvirtualenv prestoadmin
+    $ cd prestoadmin/
+    $ python setup.py develop
```
Comment (kokosing) [inline README.md]: this does not install flake8

File: README.md [hunk 2]
```diff
@@ -28,6 +28,34 @@ Comprehensive documentation can be found [here](http://prestodb.github.io/presto
...
+
+4. Create a branch for local development::
+
+    $ git checkout -b name-of-your-bugfix-or-feature
+
+   Now you can make your changes locally.
+
+5. When you're done making changes, check that your changes pass `make clean lint test`, which runs flake8 and the unit tests (which test both Python 2.6 and 2.7).
```
Comment (kokosing) [inline README.md]: Errors from lint:
```
prestoadmin/__init__.py:21:1: E402 module level import not at top of file
prestoadmin/__init__.py:22:1: E402 module level import not at top of file
prestoadmin/__init__.py:26:1: E402 module level import not at top of file
prestoadmin/__init__.py:28:1: E402 module level import not at top of file
prestoadmin/__init__.py:29:1: E402 module level import not at top of file
prestoadmin/__init__.py:31:1: E402 module level import not at top of file
prestoadmin/__init__.py:32:1: E402 module level import not at top of file
prestoadmin/fabric_patches.py:237:80: E501 line too long (80 > 79 characters)
prestoadmin/main.py:393:9: E731 do not assign a lambda expression, use a def
prestoadmin/server.py:80:80: E501 line too long (81 > 79 characters)
prestoadmin/server.py:471:80: E501 line too long (81 > 79 characters)
packaging/bdist_prestoadmin.py:112:80: E501 line too long (80 > 79 characters)
tests/product/base_product_case.py:111:80: E501 line too long (84 > 79 characters)
tests/product/base_product_case.py:116:80: E501 line too long (83 > 79 characters)
tests/product/base_product_case.py:121:80: E501 line too long (80 > 79 characters)
tests/product/base_product_case.py:122:80: E501 line too long (81 > 79 characters)
tests/product/test_server_install.py:522:80: E501 line too long (80 > 79 characters)
tests/product/standalone/presto_installer.py:66:9: E265 block comment should start with '# '
tests/product/standalone/presto_installer.py:66:80: E501 line too long (96 > 79 characters)
```

File: README.md (hunk 1 above)
Comment (kokosing) [inline README.md]: and tox

File: README.md (hunk 2 above)
Comment (cawallin) [inline README.md]: Weird. Both locally and on Jenkins lint is passing for me.

Comment (cawallin) [inline README.md]: I just updated flake8 and got these failures too. gah.

File: README.md (hunk 1 above)
Comment (cawallin) [inline README.md]: updated so that setup.py installs those things

File: README.md (hunk 2 above)
Comment (cawallin) [inline README.md]: Fixed.

**Comment by cawallin** [comment]
Fixed some of the problems in https://github.com/prestodb/presto-admin/pull/144

**Comment by kokosing** [comment]
Thanks. Looks good :)

**Pull request merged**